test/dotnet.Tests/GivenThatTheUserIsRunningDoNetForTheFirstTime.cs

```diff
@@ -15,7 +15,8 @@ namespace Microsoft.DotNet.Tests
 {
 	public class GivenThatTheUserIsRunningDotNetForTheFirstTime : TestBase
     {
-        private static CommandResult _firstDotnetUseCommandResult;
+        private static CommandResult _firstDotnetInfoUseCommandResult;
+        private static CommandResult _firstDotnetNewUseCommandResult;
```

Comment:
how about you call this _firstDotnetVerbUseCommandResult?
Same for the info, how about _firstDotnetNonVerbUseCommandResult?